### PR TITLE
#167288775 postgres setup

### PR DESCRIPTION
**Pull request by nccharles**
#### What does this PR do?
PostgreSQL integration
#### Description of Task to be completed?
This integration is to connect endpoints with database 
#### How should this be manually tested?
git clone this repo
`yarn`
`yarn run start-dev`
#### What are the relevant pivotal tracker stories?
#167288775

### CODE REVIEW

File: lcov.info [hunk 1]
```diff
@@ -0,0 +1,234 @@
+TN:
```
Comment (ekumamait) [inline lcov.info:1]: Good work @nccharles is this file necessary if not let us ignore it.

**Review by ekumamait — APPROVED** [review]
@nccharles The Travis build is failing, and you have some comments but go ahead and merge if this can be rectified in the coming PR

File: lcov.info (hunk 1 above)
Comment (nccharles) [reply lcov.info:1]: Thank you @ekumamait I will ignore it in the next PR
